src/vs/workbench/contrib/bulkEdit/browser/bulkFileEdits.ts

```diff
@@ -163,18 +163,22 @@ class DeleteOperation implements IFileOperation {
 			return new Noop();
 		}
 
-		let contents: VSBuffer | undefined;
+		let fileContent: IFileContent | undefined;
 		if (!this._undoesCreateOperation && !this.options.folder) {
 			try {
-				contents = (await this._fileService.readFile(this.oldUri)).value;
+				fileContent = (await this._fileService.readFile(this.oldUri));
```

Comment:
nit pick: braces around await not required.

Reply:
Makes sense. I will change that. Copy / paste error.